app/js/components/drugOrderEntry/addForm/AddForm.jsx

```diff
@@ -275,30 +264,6 @@ export class AddForm extends React.Component {
     return false;
   }
 
-  populateEditOrderForm = () => {
```

Comment:
any reason as to why this was removed ??

Reply:
The populate edit order can only be done from the draft by clicking the edit button then it populates the form with information of the drafted order that was clicked, so right now since there is no draft functionality this function is not used